backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5622960_sys_gh12385_Replace_BIC_SwiftCode.sql

```diff
@@ -0,0 +1,18 @@
+
+UPDATE AD_Column
+SET ColumnName ='SwiftCode'
+WHERE AD_Column_ID = 575110
+;
+
+-- 2022-01-25T08:48:53.893Z
+-- URL zum Konzept
+/* DDL */ SELECT public.db_alter_table('Revolut_Payment_Export','ALTER TABLE public.Revolut_Payment_Export ADD COLUMN SwiftCode VARCHAR(20)')
+;
+
```

Comment:
mixing DDL and updates inside one transaction is prone to fail due to pending trigger events.
please either have `commit;` between the DDL and update statements, or put them into different files

Reply:
i prefer separate files! 
committing in between it has it's own issues too